src/pages/evm/wallet/SendPage.vue

```diff
@@ -176,8 +175,8 @@ export default defineComponent({
         currencyInputIsLoading() {
             return !(this.token?.decimals && this.token?.symbol) || this.isLoading;
         },
-        loadingTransaction() {
-            return localStorage.getItem('wagmi.connected') && isCorrectNetwork() ? this.isLoading || this.configIsLoading : this.isLoading;
+        isCorrectNetwork() {
+            return true;
```

Comment:
is this correct? if so, perhaps we do not need `isCorrectNetwork`

Reply:
Nop, and I don't even know why is there !!
Removing